src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -103,7 +103,7 @@ public function setFileLinkFormat($fileLinkFormat)
      * The latter takes precedence and any output from the former is cancelled,
      * if and only if nothing bad happens in this handling path.
      */
-    public function handle(\Exception $exception)
+    public function handle(\Throwable $exception)
```

Comment:
BC break, not possible

Reply:
Will revert.

Reply:
@nicolas-grekas : which branch I can target with this change? Also, will removing the parameter type be considered a BC?

Reply:
Removing is also not allowed, see https://3v4l.org/OjTP7 and https://symfony.com/bc
We cannot do this change abruptly - it would first need a continuous upgrade path with deprecations. I don't have any other idea than deprecating this method and creating a new "handleException" one instead. For master.